kratos/processes/from_json_check_result_process.h

```diff
@@ -0,0 +1,921 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ \.
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+#if !defined(KRATOS_FROM_JSON_CHECK_RESULT_PROCESS_H_INCLUDED )
+#define  KRATOS_FROM_JSON_CHECK_RESULT_PROCESS_H_INCLUDED
+
+// System includes
+
+// External includes
+#include <unordered_map>
+
+// Project includes
+#include "processes/process.h"
+#include "includes/model_part.h"
+#include "includes/table.h"
+#include "includes/kratos_parameters.h"
+
+namespace Kratos
+{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+    /// The definition of the index type
+    typedef std::size_t IndexType;
+
+    /// The definition of the sizetype
+    typedef std::size_t SizeType;
```

Comment:
I wouldn't define these in the `Kratos` namespace.

Reply:
can't the one from the core be use here ? Should it be redefined ?